src/components/FAQ.jsx

```diff
@@ -0,0 +1,35 @@
+import { React } from "react";
+import { Link } from "react-router-dom";
+
+const FAQ = () => {
+  return (
+    <>
+      <h1 className="text-2xl font-extrabold">Hjælp og vejledning</h1>
+      <section className="bg-zinc-100 dark:bg-zinc-900 flex flex-row relative my-2 rounded font-bold p-5 flex flex-col mt-5">
+        Oplysninger om Aarhus Kommune Kultur og Borgerservices behandling af dine personoplysninger ved anvendelse af
+        Podwalk
+        <Link
+          to={"/personal-information-policy"}
+          className="text-emerald-400 dark:text-emerald-800 font-bold underline mt-5"
+        >
+          Læs hele oplysningsteksten
+        </Link>
+      </section>
+      <section className="bg-zinc-100 dark:bg-zinc-900 flex flex-row relative my-2 rounded font-bold p-5 flex flex-col mt-5">
+        Sådan bruger du navigationen
+        <Link to={"/navigation-help"} className="text-emerald-400 dark:text-emerald-800 font-bold underline mt-5">
```

Comment:
Can we use mt-3 instead, to reduces the spacing a little?

Reply:
Ofc